# Housing Market

### Introduction:

This time we will create our own dataset with fictional numbers to describe a house market. As we are going to create random data don't try to reason of the numbers.

### Step 1. Import the necessary libraries

In [40]:
import pandas as pd
import numpy as np

### Step 2. Create 3 differents Series, each of length 100, as follows:
1. The first a random number from 1 to 4
2. The second a random number from 1 to 3
3. The third a random number from 10,000 to 30,000

In [41]:
a = pd.Series(np.random.randint(1,5,size=(100,)))

In [42]:
b = pd.Series(np.random.randint(1,4,size=(100,)))

In [43]:
c = pd.Series(np.random.randint(10000,30001,size=(100,)))

### Step 3. Let's create a DataFrame by joinning the Series by column

In [44]:
df = pd.concat([a,b,c],axis=1)

### Step 4. Change the name of the columns to bedrs, bathrs, price_sqr_meter

In [45]:
df.columns = ['bedrs','bathrs','price_sqr_meter']
df.columns

Index(['bedrs', 'bathrs', 'price_sqr_meter'], dtype='object')

### Step 5. Create a one column DataFrame with the values of the 3 Series and assign it to 'bigcolumn'

In [46]:
df2 = pd.concat([a,b,c])
df2.columns = ['bigcolumn']
df2.reset_index(drop=True)

0          2
1          1
2          1
3          4
4          1
       ...  
295    21609
296    27609
297    18565
298    29459
299    10835
Length: 300, dtype: int64

### Step 6. Oops, it seems it is going only until index 99. Is it true?

In [47]:
df2.reset_index(drop=True)

0          2
1          1
2          1
3          4
4          1
       ...  
295    21609
296    27609
297    18565
298    29459
299    10835
Length: 300, dtype: int64

### Step 7. Reindex the DataFrame so it goes from 0 to 299